### PR TITLE
Fix daily build failure

### PR DESCRIPTION
**Pull request by aashikam**
# Description
$title

### CODE REVIEW

**Review by ayeshLK — APPROVED** [review]
LGTM

File: ballerina/modules/bulk/tests/bulk_csv_delete.bal [hunk 1]
```diff
@@ -18,7 +18,10 @@ import ballerina/log;
 import ballerina/test;
 import ballerina/lang.runtime;
 
-@test:AfterSuite {}
+@test:Config {
+    enable: true,
+    dependsOn: [queryXml]
```
Comment (NipunaRanasinghe) [inline ballerina/modules/bulk/tests/bulk_csv_delete.bal:23]: no need to explicitly set `enable` as true. Its true by default :) lets update similar places
```suggestion
    dependsOn: [queryXml]
```

Comment (aashikam) [reply ballerina/modules/bulk/tests/bulk_csv_delete.bal]: 3adc801afa9ac122325288623fdf0ce52ebca47b

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**